.github/actions/auto-major-minor-release-v1/action.yml

```diff
@@ -0,0 +1,168 @@
+name: Automatically create patch release every 2 weeks
+description: Triggered on schedule by our integrations to patch release
+inputs:
+  axe-version:
+    required: true
+    type: string
+  release-command:
+    required: true
+    type: string
+  release-branch:
+    required: true
+    type: string
+  default-branch:
+    required: true
+    type: string
+  token:
+    required: true
+    type: string
+  project_token:
+    required: true
+    type: string
+  slack_webhook:
+    required: true
+    type: string
+runs:
+  using: "composite"
+  steps:
+    # Unlike patch release, it is expected that the integration script does work beforehand
+    # Specifically, updating axe-core and calculating major vs minor release is done before this
+    # gets called.
+
+    - name: Cancel if we already have a PR
+      uses: dequelabs/axe-api-team-public/.github/actions/cancel-if-release-pr-exists@main
+      with:
+        token: ${{ inputs.token }}
+
+    - name: Create axe-core update commit
+      shell: bash
+      run: |
+        git config user.name "API Team CI User"
+        git config user.email "aciattestteamci@deque.com"
+        git commit -am "feat: update axe-core to v$version"
+      env:
+        version: ${{ inputs.axe-version }}
+    - name: Run preparation script
+      shell: bash
+      run: ${{ inputs.release-command }}
+    - name: Get version for later steps
+      id: get-new-version
+      shell: bash
+      run: echo version=$(node -pe 'require("./package.json").version') >> "$GITHUB_OUTPUT"
+
+    - name: Generate commit list
+      id: commit-list
+        # Multi-line output format:
+      # https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#example-of-a-multiline-string
+      shell: bash
+      run: |
+        git fetch origin "$RELEASE_BRANCH";
+        git fetch origin "$DEFAULT_BRANCH";
+        MASTER_HASH="$(git rev-parse "origin/$RELEASE_BRANCH")"
+        DEV_HASH="$(git rev-parse "origin/$DEFAULT_BRANCH")"
+        COMMITS="$(git log --oneline --no-merges --graph --abbrev-commit $MASTER_HASH..$DEV_HASH)"
+        echo "Commits:"
+        echo "$COMMITS"
+
+        echo "commits<<AUTO_RELEASE_HEREDOC" >> "$GITHUB_OUTPUT"
+        echo "$COMMITS" >> "$GITHUB_OUTPUT"
+        echo "AUTO_RELEASE_HEREDOC" >> "$GITHUB_OUTPUT"
+      env:
+        RELEASE_BRANCH: ${{ inputs.release-branch }}
+        DEFAULT_BRANCH: ${{ inputs.default-branch }}
+
+    - name: Create Docs issue
+      uses: dacbd/create-issue-action@ba4d1c45cccf9c483f2720cefb40e437f0ee6f7d
+      id: new-docs-issue
+      with:
+        token: ${{ inputs.project_token }}
+        repo: docs-devtools-html
+        assignees: enlarsen
+        title: "[TEST] ${{ github.event.repository.name }} v${{ steps.get-new-version.outputs.version }} Release Notes Needed"
```

Comment:
Are we planning on leaving in the `[TEST]` part of the title or is that just for now in order to test?

Reply:
The [TEST] is just so that people know the issues/PRs are not genuine release things. Also to make it easier to clean up. They will be removed when moving from draft